src/NISE_subs.c

```diff
@@ -143,6 +143,16 @@ time_t log_time(time_t t0, FILE* log) {
     return t1;
 }
 
+/* Compare a string to an array of options */
+int compare_string(char* string_to_compare, char* string_array[], int array_size) {
```

Comment:
This function seems to check if a string is found in an array of strings. That is not reflected by the function name which implies it is similar to `strcmp`. I would suggest renaming the function to `string_in_array`.

Reply:
Agree - changed name.